### ISSUE TITLE
parser: "babylon" is deprecated

### ISSUE THREAD

**Issue by tebuevd** [dotansimha/graphql-code-generator#2277]
**Describe the bug**
When running `graphql-codegen` I get the following warning:
`{ parser: "babylon" } is deprecated; we now treat it as { parser: "babel" }.`

**To Reproduce**
Run `graphql-codegen` version 1.4.0 with the following `codegen.json`:

3. My `codegen.yml` config file:

```json
{
  "overwrite": true,
  "schema": "./schema/schema.gql",
  "documents": null,
  "generates": {
    "./index.js": { "plugins": ["flow"] },
    "./schema.json": { "plugins": ["introspection"] }
  }
}
```

**Expected behavior**
No warning

**Environment:**

- OS: macOS 10.14.5
- `@graphql-codegen/...`: 
- NodeJS: 10.15.3

**Comment by dotansimha** [comment]
Thanks @tebuevd ! 
Fixed in https://github.com/dotansimha/graphql-code-generator/commit/06154dab62be87721e69fea0ce96481288d44bf1

**Comment by dotansimha** [comment]
Fixed in 1.5.0 🎉
